### ISSUE TITLE
Repeated fields

### ISSUE THREAD

**Issue by jernst** [kartikprabhu/mf2py#60]
TL;DR: I find repeated fields in the JSON output, and I don't know why they show up, or should show up. (This is a question more so than an "issue". All help appreciated.)

Detail: I took mf2py and put a [tiny main program](https://github.com/uboslinux/ubos-python/blob/master/mf2py-python/mf2dump.py) on top. I'm throwing this at a test Wordpress install with various IndieWeb-related plugins, in order to figure out how to fix my own site whose theme doesn't know microformats.

mf2py emits, among many other things:
```
                "photo": [
                    "http://1.gravatar.com/avatar/7a811e970877bf11f9c622d4a3b1147e?s=100&d=mm&r=g",
                    "http://1.gravatar.com/avatar/7a811e970877bf11f9c622d4a3b1147e?s=100&d=mm&r=g"
                ],
                "published": [
                    "2018-02-23T04:14:49+0000",
                    "2018-02-23T04:14:49+0000",
                    "2018-02-23T04:14:49+0000"
                ]
```
The repetitions make no sense to me. This is the URL of the "installation succeeded post" of a default Wordpress install with various Indie Web plugins and theme Independent Publisher.

When I switch themes to SemPress, and run it again, those duplicates have disappeared as far as I can see, but I'm getting:
```
                "url": [
                    "http://test1/blog/2018/02/23/hello-world/",
                    "http://test1/blog/2018/02/23/hello-world/",
                    "https://wordpress.org/"
```
in the `properties` of the `item` with type `h-entry`.

**Comment by kartikprabhu** [comment]
@jernst would be good to have the HTML this is being used on.

**Comment by kartikprabhu** [comment]
@jernst the mf2 parsed output is correct for the given HTML. The `h-entry` is on the `<body>` tag and it does indeed have three `dt-published` and two `u-photo`.
